packages/mobx/__tests__/v5/base/observables.ts

```diff
@@ -0,0 +1,18 @@
+import { observable } from "../../../src/mobx"
+
+test("argumentless observable adds undefined to the output type", () => {
```

Comment:
Currently all the TS related tests are located in the [typescript-tests.ts](https://github.com/mobxjs/mobx/blob/main/packages/mobx/__tests__/v5/base/typescript-tests.ts) file. Could you move these tests to that file?